_posts/en/posts/2016-06-06-compact-blocks-faq.md

```diff
@@ -0,0 +1,144 @@
+
+The FBRP is how the participating nodes communicate block information to each other. Nodes keep track of what transactions they send to each other, and relay block differentials based off of this knowledge. This protocol is nearly optimal for one-to-one server-client communication of new blocks. This however requires a not-well connected relay topology is more brittle than a more general p2p network. Improvements at the protocol level using compact blocks will shrink the performance gap between the curated network of nodes and the p2p network in general. The increased robustness of the p2p network and block propagation speed at large will play a role in how the network develops in the future. 
+
+## Does this scale Bitcoin?
+
+This feature is intended to save peak block bandwidth for nodes, reducing bandwidth spikes which can degrade end-user internet experience. However, the centralization pressures of mining exist in a large part due to latency of block propagation, as described in the following video. Compact blocks version 1 is not necessarily designed to solve that problem.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Y6kibPzbrIc" frameborder="0" allowfullscreen> </iframe>
+
+It is expected that miners will continue to use the [Fast Relay Network](http://bitcoinrelaynetwork.org/) until a lower-latency or more robust solution is developed. However improvements to the base p2p protocol will increase robustness in the case of FRN failure, and perhaps reduce the advantage of private relay networks, making them not worthwhile to run.
+
+Furthermore, the experiments conducted and data collected using the first version of compact blocks will inform the design of the future improvements we expect to be more competitive with the FRN.
+
+## Who benefits from compact blocks?
+
+* Full node users who want to relay transactions but who have limited internet bandwidth. If you simply want to save the most bandwidth possible while still relaying blocks to peers, there is a `blocksonly` mode already available starting in Bitcoin Core v0.12.  Blocks-only mode only receives transactions when they're included in a block, so there is no extra transaction overhead.
+
+* The network as a whole. Decreasing block propagation times on the p2p network creates a healthier network with a better baseline relay security margin. 
+
+## What is the timeline on coding, testing, reviewing and deploying compact block propagation?
+
+The first version of compact blocks has been assigned BIP152, has a working implementation, and is being actively tested by the developer community. 
+
+- BIP152: <https://github.com/bitcoin/bips/blob/master/bip-0152.mediawiki>
+- Reference implementation: <https://github.com/TheBlueMatt/bitcoin/tree/udp>
+
+## How can this be adapted for miners?
+
+In order to get miner adoption additional improvements to the compact block scheme must be made. These are two-fold:
```

Comment:
Huh? They will adopt this stuff anyway because its faster even without additional changes???